src/providers/ffz/FfzEmotes.hpp

```diff
@@ -14,9 +14,6 @@ class EmoteMap;
 
 class FfzEmotes final
 {
-    static constexpr const char *globalEmoteApiUrl =
-        "https://api.frankerfacez.com/v1/set/global";
-
```

Comment:
Unrelated change(?)

Reply:
This variable is unused.

Reply:
Doesn't impact functionality, fine by me.